testsuite/smoke-test/src/operational_tooling.rs

```diff
@@ -356,7 +356,23 @@ fn test_set_operator_and_add_new_validator() {
     let txn_ctx = op_tool
         .add_validator(validator_account, &diem_backend, false)
         .unwrap();
-    assert_ne!(VMStatusView::Executed, txn_ctx.execution_result.unwrap());
+    let execution_result = txn_ctx.execution_result.unwrap();
```

Comment:
nit: any chance we could move this verification into a new smoke test dedicated to verifying the error category and reason? The issue with the test_set_operator_and_add_new_validator() test is that it's already pretty long and difficult to reason about

Reply:
Sure thing! I'll move it into a separate one.